specter-desktop/docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
     command:
       - --host=0.0.0.0
       - --specter-data-folder=/data
+      - --devstatus-threshold=beta
```

Comment:
This change will prevent existing SWAN plugin users on Umbrel to not see the plugin after the upgrade.